rita_common/src/dashboard/wifi.rs

```diff
@@ -722,6 +743,52 @@ fn strip_string_quotes(mut name: String) -> String {
     name
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ExtenderUpdate {
+    pub device_mac: String,
+    pub wifi_info: Vec<WifiInterface>,
+    pub logging_settings: LoggingSettings,
+    pub additional_settings: ExtenderAdditionalSettings,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ExtenderAdditionalSettings {
+    pub wg_key: Option<WgKey>,
+    pub operator_addr: Option<clarity::Address>,
+}
+
+/// This is the list of settings rita extender stores internally
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ExtenderCheckin {
+    pub device_mac: String,
+    pub wifi_info: Vec<WifiInterface>,
+}
+
+impl Default for ExtenderCheckin {
+    fn default() -> ExtenderCheckin {
```

Comment:
this should be a function with a result, not default. Since default is a popular trait it will get called often even by other traits which expect it not to fail. In this case we need to be able to keep trying if we experience an error.

Reply:
fixed